src/components/Header/index.tsx

```diff
@@ -0,0 +1,252 @@
+	blank?: boolean;
+	routes: Route[];
+}
+const getNavigationRoutes = (routes: Route[]) => {
+	return routes.map((route, index) => {
+		if (route['routes']) {
+			return (
+				<NavigationItem pl={[0, 0, 3, 3]} key={index} mx={15}>
+					<NavDropDown title={route.title} showDividers={route.showDividers}>
+						{getNavigationRoutes(route.routes)}
+					</NavDropDown>
+				</NavigationItem>
+			);
+		}
+
+		return (
+			<NavigationItem pl={[0, 0, 3, 3]} key={index} mx={[2, 2, 15]}>
+				<Link
+					blank={route.blank || false}
+					href={route.path}
+					style={{
+						padding: '3px 0 3px',
+						display: 'auto',
+					}}
+					color="text.main"
+					fontSize={13}
+				>
+					<Txt.span bold>{route.title}</Txt.span>
+				</Link>
+			</NavigationItem>
+		);
+	});
+};
+
+export const Wrapper = styled.div`
+	position: relative;
+	width: 100%;
+	background: white;
+	z-index: 5;
+
+	+ div:first-of-type {
+		margin-top: 80px;
+	}
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		box-shadow: 0px 0px 21px 5px rgba(152, 173, 227, 0.2);
+	}
+`;
+
+const Actions = styled(Box)`
+	justify-content: flex-end;
+	align-items: center;
+`;
+
+const MobileActions = styled(Flex)`
+	display: none;
+	min-height: 84px;
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		display: flex;
+	}
+`;
+
+const NavigationItems = styled(Flex)<{ menuOpen?: boolean }>`
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		top: 0;
+		right: 0;
+		width: 300px;
+		bottom: 0;
+		justify-content: flex-start;
+		transform: translateX(100%);
+		transition: transform 200ms ease-in-out;
+		padding: 16px;
+		overflow: auto;
+		flex-wrap: nowrap;
+
+		${(props) =>
+			props.menuOpen
+				? css`
+						position: fixed;
+						transform: translateX(0%);
+				  `
+				: ''}
+	}
+`;
+
+export const Navbar = styled(Flex)`
+	min-height: 80px;
+	position: relative;
+	flex-direction: column;
+	justify-content: center;
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		min-height: auto;
+		padding-top: 16px;
+		padding-bottom: 16px;
+	}
+`;
+
+export const Header = (props: HeaderProps) => {
+	const [mobileMenuOpen, setMobileMenuOpen] = React.useState(false);
+
+	const closeMenu = () => {
+		setMobileMenuOpen(false);
+	};
+	const openMenu = () => {
+		setMobileMenuOpen(true);
+	};
+	return (
+		<Wrapper>
+			<Container px={[3, 3, 3, 0]}>
+				<Navbar>
+					<Flex justifyContent="space-between" alignItems="center" width="100%">
+						<Flex alignItems="center">
+							<Link href={props.rootRoute} style={{ lineHeight: 1 }}>
+								<Img alt={props.logoAltText} src={props.logo} height="42" />
+							</Link>
+						</Flex>
+						{mobileMenuOpen && (
+							<Fixed
```

Comment:
Can you check out the `Layer` grommet component? That's what we use for the modal, and it might be usable for the drawer as well. Speaking of which, the mobile option is a drawer, so we can define it as such (we can only have it internal for now)